frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -388,6 +388,90 @@ export class ChallengesettingsComponent implements OnInit, OnDestroy {
     SELF.globalService.showModal(PARAMS);
   }
 
+    /**
```

Comment:
Formatting seems a bit off in this file, can we please fix it?

Reply:
Fixed it!